packages/vuetify/src/components/VTable/VTable.sass

```diff
@@ -129,6 +129,8 @@
 
   .v-table--has-bottom
     > .v-table__wrapper
+      border-bottom-left-radius: 0
+      border-bottom-right-radius: 0
       > table
         > tbody
           > tr
```

Comment:
The rest of this is all redundant, those elements never had border radius to override in the first place.